src/view/transaction-detail-mobile.js

```diff
@@ -62,45 +53,44 @@ const TransactionDetailView = ({ store, nav }) => (
         <Title title="Transaction Detail" />
         {store.selectedTransaction.type === 'bitcoin' ? (
           <View style={[styles.icon, styles.btc]}>
-            <BitcoinIcon height={25} width={20} />
+            <BitcoinIcon height={135 * 0.19} width={170 * 0.19} />
```

Comment:
How do you come to these formulas? I'm fine to do these changes myself so you don't have to refactor them in, but I'm not sure how you arrive at them :)

Reply:
I take the base height/width from the svg and kinda just play around with a scaling factor that looks good :D